insights/tests/client/connection/test_checkin.py

```diff
@@ -24,6 +24,7 @@ def _get_canonical_facts_response(canonical_facts):
 @patch('insights.client.connection.InsightsUploadConf.get_rm_conf', return_value={})
```

Comment:
I rêmembered! `get_rm_conf` is called from the depths of `get_canonical_facts`. Getting the canonical facts is not as trivial as it might sound like and it initializes a Kraken-sized component.